src/UnitTests/Bug/ExpressionMapping.cs

```diff
@@ -260,14 +260,14 @@ public void When_Using_Non_TypeMapped_Class_Method()
             _valid = new Parent { DateTime = DateTime.Now };
         }
 
-        [Fact]
+        [Fact(Skip = "Failing test")]
```

Comment:
Ok this is all my fault.  I accept full responsibility for this failure.

Replace lines 264 and 274 with `_valid = new Parent { DateTime = new DateTime(2015,1,1) };` to have the tests past, for this year and every other year.